automation/az_elev_chart.py

```diff
@@ -0,0 +1,210 @@
+        If this parameter is set to true then elevation is scaled for use full
+        area. It is easier to assess the shape of the chart. Information about
+        scalling is placed in legend.
+        If this parameter is set to false then we draw auxiliary line Y=90 degree.
+    axis_in_local_time: bool
+        If true then time axis (X axis in azimuth/elevation from time plot) is 
+        in local time. Otherwise in UTC.
+    scale_polar: float
+        @width and @height parameters will scale by this factor for plot polar
+        chart. 
+
+    Returns
+    =======
+    Plot azimuth/elevation polar chart and next azimuth/elevation from time chart.
+    '''
+
+    date_series, azimuth_series, elevation_series = _calculate_series(sat_id, aos, los, location, time_step)
+    _plot_polar_azimuth_elevation(azimuth_series, elevation_series,
+        int(width * scale_polar), int(height * scale_polar))
+    _plot_azimuth_and_elevation_from_time(date_series, azimuth_series, elevation_series,
+        width=width, height=height,
+        scale_elevation=scale_elevation, axis_in_local_time=axis_in_local_time)
+
+
+def _plot_azimuth_and_elevation_from_time(date_series: Sequence[datetime.datetime],
+        azimuth_series: Sequence[float], elevation_series: Sequence[float],
+        width=50, height=20, scale_elevation=True, axis_in_local_time=True):
+    '''
+    Plot azimuth/elevation from time chart.
+    Series colors are compatible with gPredict.
+    '''
+    # We scale elevation, because plotille library allow draw only one Y axis.
+    if scale_elevation:
+        elevation_series = [el *4 for el in elevation_series]
+    if axis_in_local_time:
+        target_tz = tz.tzlocal()
+        date_series = [d.astimezone(target_tz) for d in date_series]
+
+    if len(date_series) < 2:
+        print("No plot. Not enough data.")
+        return
+
+    fig = plotille.Figure()
+    fig.width = width
+    fig.height = height
+    fig.color_mode = 'byte'
+    fig.set_y_limits(0, 360)
+    fig.plot(date_series, azimuth_series, label="Azimuth", lc=COLOR_BLUE)
+    elevation_label = "Elevation"
+    if scale_elevation:
+        elevation_label += " * 4"
+    else:
+        fig.plot(date_series, [90,] * len(date_series), interp=None, label="Y=90")
+    fig.plot(date_series, elevation_series, label=elevation_label, lc=COLOR_RED)
+    fig.y_label = "Degrees"
+    x_label = "Time (%s)" % ("local" if axis_in_local_time else "UTC",) 
+    fig.x_label = x_label
+    print(fig.show(legend=True))
+
+def _plot_polar_azimuth_elevation(azimuth_series: Sequence[float], elevation_series: Sequence[float],
+        width=50, height=20):
+    '''
+    Plot azimuth/elevation polar chart.
+    Chart has auxiliary lines on -90, -60, -30, 30, 60, 90 degrees.
+    AOS point is highlighted in red.
+    '''
+    # Max values is increased by 0.05. Without it points with any coordinate
+    # eqauls max were not drawed.
```

Comment:
```suggestion
    # eqauls max were not drawn.
```

draw is an inrregular verb.